core/actionresult/src/main/kotlin/nl/q42/template/actionresult/data/ActionResultMapper.kt

```diff
@@ -3,82 +3,98 @@ package nl.q42.template.actionresult.data
 import com.haroldadmin.cnradapter.NetworkResponse
 import com.squareup.moshi.JsonEncodingException
 import io.github.aakira.napier.Napier
-import kotlinx.coroutines.CancellationException
-import nl.q42.template.actionresult.domain.ActionResult
 import java.io.EOFException
 import java.io.IOException
 import java.net.ConnectException
 import java.net.SocketTimeoutException
 import java.net.UnknownHostException
+import kotlinx.coroutines.CancellationException
+import nl.q42.template.actionresult.domain.ActionResult
 
 private const val UNAUTHENTICATED_STATUSES = 401
 private const val NOT_FOUND_REQUESTS_STATUS = 404
 private const val TOO_MANY_REQUESTS_STATUS = 429
 
 private fun isUnAuthorized(httpStatusCode: Int?) = httpStatusCode == UNAUTHENTICATED_STATUSES
+
 private fun isNotFound(httpStatusCode: Int?) = httpStatusCode == NOT_FOUND_REQUESTS_STATUS
+
 private fun isTooManyRequests(httpStatusCode: Int?) = httpStatusCode == TOO_MANY_REQUESTS_STATUS
 
 /**
  * Maps a remote [NetworkResponse] to an [ActionResult] transforming all possible errors and exceptions into our domain error models
  */
-suspend fun <T : Any> mapToActionResult(request: suspend () -> NetworkResponse<T, ApiErrorResponse>): ActionResult<T> =
-    try {
-        request().networkResponseToActionResult()
-    } catch (e: Exception) {
-        if (e is CancellationException) ActionResult.Error.Cancelled(e) else ActionResult.Error.Other(e)
+suspend fun <T : Any> mapToActionResult(request: suspend () -> NetworkResponse<T, ApiErrorResponse>): ActionResult<T> = try {
+    request().networkResponseToActionResult()
+} catch (e: Exception) {
+    if (e is CancellationException) {
+        ActionResult.Error.Cancelled(
+            e
```

Comment:
Is this rule "arguments on newlines, always"? Not a great result here

Reply:
oh I think it's the line length limit

Reply:
I don't think it's the max line length, but maybe it's a matter of playing with the settings to find the ones that go well with our preferences and set it as the standard for the team/Q42